examples/cov3d_simulation.py

```diff
@@ -0,0 +1,128 @@
+"""
+This script illustrates the example of Covariance 3D estimation using simulation images
+generated from Gaussian blob volumes.
+"""
+import logging
+import numpy as np
+from scipy.cluster.vq import kmeans2
+
+from aspire.source.simulation import Simulation
+from aspire.basis.fb_3d import FBBasis3D
+from aspire.utils.filters import RadialCTFFilter
+from aspire.estimation.noise import WhiteNoiseEstimator
+from aspire.estimation.mean import MeanEstimator
+from aspire.estimation.covar import CovarianceEstimator
+from aspire.utils.matlab_compat import Random
+from aspire.utils.matrix import eigs
+from aspire.utils.misc import src_wiener_coords
+
+
+logger = logging.getLogger('aspire')
+
+# Specify parameters
+num_vols = 2           # number of volumes
+img_size = 8           # image size in square
+num_imgs = 1024        # number of images
+num_eigs = 16          # number of eigen-vectors to keep
+
+# Create a simulation object with specified filters
+sim = Simulation(
+    n=num_imgs,
+    C=num_vols,
+    filters=[RadialCTFFilter(defocus=d) for d in np.linspace(1.5e4, 2.5e4, 7)]
+)
+
+# Specify the normal FB basis method for expending the 2D images
+basis = FBBasis3D((img_size, img_size, img_size))
+
+# Estimate the noise variance. This is needed for the covariance estimation step below.
+noise_estimator = WhiteNoiseEstimator(sim, batchSize=500)
+noise_variance = noise_estimator.estimate()
+logger.info(f'Noise Variance = {noise_variance}')
+
+"""
```

Comment:
Why are the comments sometimes written with `#` and sometimes with `"""`? We should be consistent.

Reply:
Generally it is more convenient to use `"""` for multi-line comments. Changed to make consistent in the new commit.